grid2op/multi_agent/utils.py

```diff
@@ -74,4 +78,77 @@ def __eq__(self, other):
             and self._current_agent == other._current_agent
             and self.selected_agent == other.selected_agent
         )
-        
\ No newline at end of file
+
+
+class ClusterUtils:
+    """
+    Utility class for clustering substations
```

Comment:
Suggestion: """ Outputs clustered substation based on the Louvain graph clustering method"""